src/api/app/models/tokens/base.rb

```diff
@@ -1,23 +1,20 @@
-class Token < ApplicationRecord
-  belongs_to :user, foreign_key: 'user_id', inverse_of: :tokens
+class Tokens::Base < ApplicationRecord
```

Comment:
I dont think it makes sense to use `Tokens::Base` here? Why not just call the base class `Token`? Since we should be able to do stuff like `Token.all` or `Token.find_by_string()` etc.